[zilliqa] Add Kalijo (SEED)

darthgus opened the pull request:
### Adding a token

- [x ] Create a folder in the chain assets directory named after your contract address
- [x ] Your logo must be either SVG or PNG min 100x100 max 500x500 and named `logo.<extension>`
- [x ] Your logo should be a square (same width and height)
- [x ] Use your best judgement to see if you need to add padding or background transparency depending on your design
- [x ] A PNG should **always** use tinypng.com to compress it
- [x ] An SVG should use svgminify.com to remove useless props and be the same width/height ratio. Do not embed PNG data in them, use a PNG instead.
- [x ] Logo should be < 20kb
- [x ] Add information of your project in `meta.json`, **USE** jsonlint.com to check validity following the spec from the README.

Make sure your meta [JSON is valid](https://jsonlint.com) at all times before submitting a PR and conforms to the spec.
PRs will be closed without notice if failing to follow these simple instructions.

darthgus commented:
Hi @balthazar just wondering if you can review and merge my PR please?

balthazar commented:
Can you fix the svg dimensions in order to be square?

darthgus commented:
hey @balthazar changed to png square.  Is this ok?

balthazar commented:
I would recommend to keep the SVG since its infinitely scalable, actually I see your viewport was correct at 400x400 so not sure why my render wasnt a square. Can you revert your png? Sorry for the trouble

darthgus commented:
Hey @balthazar i have reverted - yes we (thought) we had made the SVG square! Couldn't work out what else we could do there, so we went the PNG route. But that's now reverted.
